### PR TITLE
fix(BCH): Check if bch change index got behind btc by gap limit

### PR DESCRIPTION
**Pull request by plondon**

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/data/modules/addressesBch/sagas.js [hunk 1]
```diff
@@ -24,7 +28,54 @@ export default ({ coreSagas }) => {
...
+      index,
+      bchChangeIndex,
+      btcChangeIndex,
+      state
+    )
+
+    // if there are addrs to display show them else carry on
+    /* eslint-disable */
+    if (addrs.length) {
+      window.alert(
```
Comment (tony-blockchain) [inline packages/blockchain-wallet-v4-frontend/src/data/modules/addressesBch/sagas.js]: nice solution although these might be blocked by browser, console.log is safer imo

Comment (plondon) [reply packages/blockchain-wallet-v4-frontend/src/data/modules/addressesBch/sagas.js]: We're a bit worried users won't be able to easily open the console. I'll check in with support after the fix is deployed, if anyone has issues we'll switch to console log.

File: packages/blockchain-wallet-v4-frontend/src/scenes/Settings/Addresses/Bch/Wallets/template.success.js [hunk 2]
```diff
@@ -160,7 +161,16 @@ const Success = props => {
                     defaultMessage='Show xPub'
                   />
                 </ClickableText>
-              )
+              ),
+              <ClickableText
+                size='small'
+                onClick={() => onShowChangeAddrs(wallet.value)}
```
Comment (tony-blockchain) [inline packages/blockchain-wallet-v4-frontend/src/scenes/Settings/Addresses/Bch/Wallets/template.success.js]: onClick={onShowChangeAddrs.bind(null, wallet.value)}

File: packages/blockchain-wallet-v4-frontend/src/scenes/Settings/Addresses/Bch/Wallets/template.success.js [hunk 3]
```diff
@@ -160,7 +161,16 @@ const Success = props => {
                     defaultMessage='Show xPub'
                   />
                 </ClickableText>
-              )
+              ),
+              <ClickableText
```
Comment (tony-blockchain) [inline packages/blockchain-wallet-v4-frontend/src/scenes/Settings/Addresses/Bch/Wallets/template.success.js]: I think we should use selector to determine the difference between btc and bch change addresses and only show link in this case

**Pull request merged**